docker/jenkins/check_if_tag.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
 # Used to trigger downstream Jenkins jobs
-TRIGGER_FILE=.commit_is_tag
+TRIGGER_FILE=".commit_is_tag"
 rm -rf $TRIGGER_FILE
 
-if git describe --tags --exact-match $GIT_COMMIT 2> /dev/null;
-then
+TAG=$(git describe --tags --exact-match $GIT_COMMIT 2> /dev/null)
+if [[ -n "$TAG" ]] && [[ "$TAG" =~ ^[0-9]{4}-[0-9]{2}-[0-9]{2}(\.[0-9])?$ ]]; then
```

Comment:
Taken from the [standup project](https://github.com/mozilla/standup/blob/master/bin/deploy.sh#L11).